flask/common/src/main/java/net/corda/flask/common/Flask.java

```diff
@@ -0,0 +1,89 @@
+package net.corda.flask.common;
+
+import lombok.SneakyThrows;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.AbstractMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.zip.CRC32;
+import java.util.zip.ZipEntry;
+
+public class Flask {
+
+    public static class Constants {
+        public static final String DEFAULT_LAUNCHER_NAME = "net.corda.flask.launcher.Launcher";
+        public static final String LIBRARIES_FOLDER = "LIB-INF";
+        public static final String METADATA_FOLDER = "META-INF";
+    }
+
+    public static class ManifestAttributes {
+        public static final String APPLICATION_CLASS = "Application-Class";
+        public static final String JVM_ARGS = "JVM-Args";
+        public static final String JAVA_AGENTS = "Java-Agents";
+        public static final String ENTRY_HASH = "MD5-Digest";
+    }
+
+    public static class JvmProperties {
+        /**
+         * If this property is set, its value will be appended to the jvm arguments
+         * list of the spawned Java process
+         */
+        public static final String JVM_ARGS = "net.corda.flask.jvm.args";
+
+        /**
+         * This JVM property can be used to override the path to the flask cache folder
+         */
+        public static final String CACHE_DIR = "net.corda.flask.cache.dir";
+
+        /**
+         * This JVM property can be used to always wipe the library cache directory before program startup
+         */
+        public static final String WIPE_CACHE = "net.corda.flask.cache.wipe";
+    }
+
+    @SneakyThrows
```

Comment:
the thing is that I have no intention of handling such an exception, I'm perfectly fine with it crashing the whole process (I cannot do much anyway since an `IOException` in this part of the code is an unrecoverable condition). For this reason I find the code to handle the exception (that vanilla Java forces you to write) to be just noise